### ISSUE TITLE
Batch Normalization

### ISSUE THREAD

**Issue by HosnaCSE** [carpedm20/DCGAN-tensorflow#247]
Hi,
Is there anyone who tried Batch Normalization in this code or any GAN code? Does that increase the performance of accuracy?

Any kind of guidance will be highly appreciatable.
Thanks
Hosna
